.gitignore

```diff
@@ -1,6 +1,7 @@
 package-lock.json
 .DS_Store
 node_modules
+vsf-test
```

Comment:
Why is it here? :)

Reply:
I added test project in packages by mistake and then tried to remove it, but it shouldn't be here. Thanks for let me know :+1: